scripts/Firstboot-Stage/S02-logging

```diff
@@ -3,9 +3,14 @@ log "Starting S02-logging..."
 log "======================="
 
 #=============================================
-# 3) start syslog daemon
+# 3) start syslog daemon (when required)
 #---------------------------------------------
-/usr/bin/systemctl start syslog.socket
+if [[ $(/usr/bin/systemd-detect-virt) == 'wsl' ]];
```

Comment:
My bash knowledge is deeply buried so I better ask: what happens if `/usr/bin/systemd-detect-virt` is not there? Will this run the "else" part?

Reply:
Theoretically, yes. it'd evaluate to 'something' (either "" or an error message) that isn't "wsl".

Both "/usr/bin/systemd-detect-virt" and "/usr/bin/systemctl" are provided by the "systemd" package - so "systemd-detect-virt" will always be present if the (original) expectation was to run a "systemctl" command [to start syslog.socket].